### ISSUE TITLE
Test error: Cannot find the Button's aria-disabled attribute

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly#7019]
**Please describe the issue**

2 tests cannot find the Button's `aria-disabled` attribute. This test worked with v5, but no longer works with v6.

**Are there visuals for this issue? Please provide screenshots**

Error 1
![Screenshot 2024-08-28 at 6 31 05 PM](https://github.com/user-attachments/assets/036a4bf5-ad61-44c1-a699-fba0a7a028b1)

Error 2
![Screenshot 2024-08-28 at 6 24 00 PM](https://github.com/user-attachments/assets/e03631d4-7e3c-4221-81f4-e74ff65d702b)

**Could you point to a branch or draft PR where this issue exists? Please provide a link to the code**

Error 1
- Test https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.test.tsx#L186
- Button https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.tsx#L63

Error 2
- Test https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.test.tsx#L186
- Button https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.tsx#L63
